server/src/mds3.preInit.ts

```diff
@@ -20,17 +22,22 @@ export async function preInit(ds: Mds3): Promise<any> {
 	try {
 		// first try is not on a loop
 		const response = await ds.preInit.getStatus()
-		if (response?.status !== 'OK') {
-			console.log(`gdc api /status:`, response)
-			throw new Error('status is not OK: ' + response.message )
-		}
+		if (response?.status !== 'OK') throw response
 		return response
-	} catch (error: any) {
-		// TODO: should this 
+	} catch (response: any) {
+		// may retry depending on ds.preInit configuration
+		if (response.status != 'recoverableError') {
+			// should not retry on fatal error
+			console.log(`fatal error: ${ds.label} ds.preInit.getStatus()`, response)
+			throw new Error('status is not OK: ' + response.message)
+		}
+
 		if (retryMax < 1) {
-			throw new Error('GDC API status error: ' + (error.message || error))
+			console.log('35 mayRetryDsPreInit()', response)
+			throw new Error('GDC API status error: ' + (response.message || response))
```

Comment:
if ds.preInit() is intended to be generic but not gdc-specific, may delete mentioning of GDC